_meta/beat.yml

```diff
@@ -49,9 +49,27 @@ apm-server:
     # To enable real user monitoring (RUM) support set this to true.
     #enabled: false
 
+    #-- v1 RUM endpoint
+
     # Rate limit per second and IP address for requests sent to the RUM endpoint.
     #rate_limit: 10
 
+    #-- v2 RUM endpoint
+
+    #event_rate:
+      # Defines the maximum amount of events allowed to be sent to the APM Server v2 RUM
+      # endpoint per ip per second. There is an internal burst queue to avoid rejecting short termed spikes. The burst 
+      # queue is set to limit*5.
+      # Defaults to 1000.
+      #limit: 1000
```

Comment:
sorry to keep discussing defaults, but i think 1000 events per second is too high. For reference, the previous discussion is here: https://github.com/elastic/apm-server/pull/1367#discussion_r218004645

following the previous discussing, what about a limit of 500 and a burst of 5? And I think even that is high. It means each browser can make a new request with the maximum number of spans per second - and most websites takes way longer than 1s to load.